ara_plumes/regressions.py

```diff
@@ -444,6 +427,8 @@ def kernel_density_fit(param_hist, bandwidth=1):
     return param_opt_canidates, kde_models
 
 
+# OLD function
+# Delete DEAD CODE
```

Comment:
[YAGNI](https://martinfowler.com/bliki/Yagni.html).  Go ahead and delete it if it's not being used in this package (and it's not being used in plumex).  If you think it might be useful, you can always get it back with: 
```
git log --oneline -S <search_string> -- ara_plumes/regressions.py
```
followed by appropriate `git show`.

Reply:
whoops! forgot to remove.